tools/zeek-to-vast/README.md

```diff
@@ -0,0 +1,50 @@
+# zeek-to-vast
+
+The **zeek-to-vast** tool enables [Zeek](https://zeek.org) to speak with
+[VAST](http://vast.io) via [Broker](https://github.com/zeek/broker). That is,
+it acts as bridge between Zeek and VAST:
+
+                  Zeek  <--->  zeek-to-vast  <--->  VAST
+
+Note that `zeek-to-vast` acts as a "dumb" relay and does not contain much logic.
+Please consult the documentation of the Zeek package [zeek-vast][zeek-vast] for
+concrete Zeek use cases.
+
+## Installation
+
+First, make sure you have Zeek and Broker installed. VAST automatically builds
+`zeek-to-vast` if Broker is found during the build configuration. If you're
+using the `configure` script in this repo, the flag `--with-broker=PATH` allows
+for specifying a custom install location. `PATH` is either an install prefix or
+a build directory of a Broker repository.
+
+Second, install the Zeek scripts via:
+
+    zeek-pkg install zeek-vast
```

Comment:
```suggestion
    zkg install zeek-vast
```

Reply:
[It looks like](https://github.com/zeek/zeek/issues/241) it will be called `zkg`.